tests/api/serve.js

```diff
@@ -111,6 +111,13 @@ export default async (request, response) => {
     console.log(`LOCAL:    serving local file: ${filePath}`);
     const output = JSON.parse(await fs.readFile(filePath));
 
+    // If the bundle contains a `now` key, override any
+    // configured value for the current time with the timestamp
+    // present in the bundle
+    if(output["@bundle"]?.now){
+      config.now = output["@bundle"].now;
+    }
+
```

Comment:
Here's where we read and set the "now" time from the bundle metadata, if present